### DISCUSSION TITLE
Guardian HP bar estimate

### DISCUSSION THREAD

**Discussion by eyymily** [shalzuth/LostArkLogger#35] [Ideas]
something for consideration - an approximated guardian raid hp bar, using datamined values to start off and subtracting group damage dealt in real time

**Comment by shalzuth** [comment]
Will not happen in the near future. Smilegate purposefully excludes that data, so we will exclude it as well.

**Reply by drhouse** [reply]
just curious, is the reason this won't be implemented due to difficulty or the TOS violation? I could understand the former, since one would probably need to have everyone in a group using ACT to total the dps. If it's the latter, given that the existence of LostArkLogger is already frowned upon by Smilegate, would it matter if guardian hp bar were also added?

**Reply by darrendlin** [reply]
i don't think it makes sense to argue "if we're already technically breaking rules, it's ok if we break more rules".

**Reply by drhouse** [reply]
hypothetically, if the rules being broken were similar, then it would make sense according to consistency
